projects/core/styles/mixins/textfield.less

```diff
@@ -145,6 +145,13 @@
     color: var(--tui-text-02);
     pointer-events: none;
 
+    /* Safari 9+ */
+    @supports (-webkit-hyphens: none) {
```

Comment:
should we duplicate it for `projects/core/styles/mixins/textfield.scss` too ?

Reply:
updated